src/styles/theme.js

```diff
@@ -0,0 +1,35 @@
+export default {
+  colors: {
+    light: 'rgb(255, 255, 255)',
+    dark: 'rgb(25, 25, 25)',
+    semiDark: 'rgba(25, 25, 25, 0.5)',
+    semiGray: 'rgba(234, 234, 234, 0.4)',
+    border: 'rgba(40, 40, 40, 0.3)'
+  },
+  globals: {
+    radius: '5px'
+  },
+  fonts: {
+    families: {
+      sourceSans: "'Source Sans Pro', sans-serif",
+      sourceCode: "'Source Code Pro', monospace"
+    },
+    sizes: {
+      regular: '0.875em',
+      medium: '1.125em',
+      large: '1.375em'
+    }
+  },
+  sizes: {
+    tiny: '0.313em',
+    small: '0.5em',
+    regular: '1em',
+    large: '1.875em',
+    huge: '5.625em',
+    enormous: '7.5em',
+    wrappers: {
+      small: '430px',
+      regular: '1110px'
+    }
+  }
```

Comment:
I never do that but I guess I can try this out with Zola :p

Reply:
😉